README.md

```diff
@@ -438,6 +438,120 @@ const clientWithNoCache = await Client.create(wallet, {
 })
 ```
 
+## Group chat
+
+![Status](https://img.shields.io/badge/Feature_Status-Pre--Preview-red)
+
+> **Important:**  
+> This feature is in **Pre-Preview** status and ready for you to start experimenting with. We do **not** recommend using Pre-Preview features in production apps. Software in this status will change as we iterate based on feedback.
+
+Use the information in this section to **experiment** with providing group chat in your app.
+
+### Enable group chat for your Client
+
+The first step is to enable group chat for your Client:
+
+```ts
+const creatorClient = await Client.create(yourSigner)
+creatorClient.enableGroupChat()
+```
+
+This enables the following capabilities required to provide group chat:
+
+- The client will be able to create group chats
+- Group chats will be present in `client.conversations.list()`
+- The client will understand GroupChat codecs such as `GroupChatMemberAdded` and `GroupChatTitleChanged`
+
+### Create a group chat
+
+Enable a user to create a group chat using `newGroupConversation` and adding member addresses to it:
+
+```ts
+const memberAddresses = [
+  '0x194c31cAe1418D5256E8c58e0d08Aee1046C6Ed0',
+  '0x937C0d4a6294cdfa575de17382c7076b579DC176',
+]
+const groupConversation =
+  creatorClient.conversations.newGroupConversation(memberAddresses)
+```
+
+Assuming the other members of the group chat have clients with group chat enabled, they'll see the new group chat in their conversation list.
+
+### Send a message to a group chat
+
+Enable a user to send a message to a group chat the same way you send messages to a 1:1 conversation:
+
+```ts
+await groupConversation.send('hello everyone')
+```
+
+### Load group chats
+
+When you enabled group chat for your Client, you enabled group chats to be returned in `conversations.list()`:
+
+```ts
+const conversations = await creatorClient.conversations.list()
+const conversation = conversations[0]
+
+console.log(conversation.isGroup) // => true when it's a group conversation
+```
+
+### Change the group chat title
+
+Enable a member of a group chat to change the group chat title by sending a message with the `GroupChatTitleChanged` content type:
+
+```ts
+import { ContentTypeGroupChatTitleChanged } from '@xmtp/xmtp-js'
+import type { GroupChatTitleChanged } from '@xmtp/xmtp-js'
+
+const
+```
+
+### Manage group state with the `GroupChat` class
+
+Use the `GroupChat` class to keep track of group state, such as the group title and member list:
+
+```ts
+const conversations = await creatorClient.conversations.list()
+const conversation = conversations[0] // assume this conversation is a group conversation
+
+const groupChat = new GroupChat(creatorClient, conversation)
+```
+
+You can also use the `GroupChat` class to change the group chat title and member list.
+
+#### Change a group chat title
```

Comment:
Is this meant to be about changing the group title or the group chat title? I ask because in https://github.com/xmtp/xmtp-js/pull/388/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R499 we talk about how to change the group chat title... 

Are these the same concept - or different? Change group title vs group chat title?

Reply:
Same thing, sorry I was inconsistent in my terminology here.